src/components/FilterButtons/FilterButtons.test.tsx

```diff
@@ -3,13 +3,18 @@ import userEvent from "@testing-library/user-event";
 import FilterButtons from "./FilterButtons";
 
 describe("Testing initial rendering", () => {
+  const setFilterNameMock = jest.fn();
   it("'Show all' button selected by default", () => {
-    const { getByText } = render(<FilterButtons />);
+    const { getByText } = render(
+      <FilterButtons setFilterName={setFilterNameMock} />
```

Comment:
Ho aggiunto la prop setFilterName in quanto Eslint mi segnalava la mancanza. Mi sembra poco corretto rendere la prop non obbligatoria solo per i test

Reply:
Sono d'accordo, se il componente non funziona senza, dovrebbe essere obbligatoria!